utils/networks.ts

```diff
@@ -18,34 +23,124 @@ export const networks = {
     railgunNetworkName: NetworkName.Ethereum,
     chainId: mainnet.id,
     wethAddress: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    evmGasType: EVMGasType.Type2,
+    baseToken: {
+      symbol: "ETH",
+      name: "Ether",
+    },
+    fallbackProviders: {
+      chainId: mainnet.id,
+      providers: [
+        {
+          provider: "https://cloudflare-eth.com/",
```

Comment:
Should we parameterize all these RPC URLs as env vars or something? For a production app we probably don't want to rely on public RPCs since they're often flaky/rate-limited

Reply:
tracking in #55